deployments/cardano-prod.nix

```diff
@@ -6,13 +6,24 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["env:production"];
 
     # DEVOPS-64: disable log bursting
     services.journald.rateLimitBurst = 0;
 
     # Initial block is big enough to hold 3 months of transactions
     deployment.ec2.ebsInitialRootDiskSize = mkForce 700;
+
+    services.dd-agent.tags = ["env:production"];
+    services.dd-agent.processConfig = ''
+      init_config:
+
+      instances:
+      - name: cardano-node
+        search_string: ['cardano-node']
+        exact_match: False
+        thresholds:
+          critical: [1, 2]
```

Comment:
This is an [inclusive range](https://github.com/DataDog/integrations-core/blob/fe6d775dbac8ba16f209b3a8abef08d9e464240b/process/check.py#L413-L414) and therefore allows 2 `cardano-node`. Do we ever expect multiple `cardano-node` processes per host?